Maestro/subscriptions.json

```diff
@@ -160,6 +160,27 @@
             "/verbosity:Normal"
           ]
         },
+        // This handler will bring the Latest CoreFX release/1.1.0 build into CoreCLR release/1.1.0
```

Comment:
The CoreCLR tests currently depend on the latest CoreFX build. These aren't just product dependencies but any time versions from one build's packages are used in another repo.